recipes/default.rb

```diff
@@ -261,6 +261,21 @@
     mode 0750
     action :create
   end
+
+   # template all the ddl files from all versions
+   cookbook_file "#{theDomain}/flyway/all/sql/V#{version}__hopsworks.sql" do
+    source "sql/ddl/updates/#{version}.sql"
+    owner node['glassfish']['user']
```

Comment:
Explicitly set group of file

Reply:
Ok, done. However, I was keeping it the same as other blocks in the recipe https://github.com/logicalclocks/hopsworks-chef/blob/master/recipes/default.rb#L205

Reply:
I'm not sure if the assigned group would be the primary group of the `owner` or the primary group of the user which runs karamel